### PR TITLE
[neuralnet] add log about training

### PR DESCRIPTION
**Pull request by lhs8928**
- Added log when start/finish training
 - Added log when get current epoch is called

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

File: nntrainer/models/neuralnet.cpp [hunk 1]
```diff
@@ -774,6 +782,11 @@ int NeuralNetwork::train_run(std::function<bool(void *userdata)> stop_cb) {
                 << " - Training Loss: " << stat.loss;
       ml_logi("# %d / %d - Training Loss: %f", epoch_idx, getEpochs(),
               stat.loss);
+      ml_logd("[NNTrainer] Training epoch %d / %d finished successfully.",
+              epoch_idx, getEpochs());
+    } else {
+      ml_logd("[NNTrainer] Training stopped unexpectedly during epoch %d.",
+              epoch_idx);
```
Comment (jijoongmoon) [inline nntrainer/models/neuralnet.cpp:789]: I wonder why it is "unexpectedly"

File: nntrainer/models/neuralnet.cpp [hunk 2]
```diff
@@ -113,6 +113,11 @@ int NeuralNetwork::loadFromConfig(const std::string &config) {
   return ML_ERROR_NONE;
 }
 
+unsigned int NeuralNetwork::getCurrentEpoch() {
```
Comment (jijoongmoon) [inline nntrainer/models/neuralnet.cpp:116]: If application developer need this ```getCurrentEpoch()```, then we need to add C/C++ API for this.

File: nntrainer/models/neuralnet.cpp [hunk 3]
```diff
@@ -113,6 +113,11 @@ int NeuralNetwork::loadFromConfig(const std::string &config) {
...
+  ml_logd("[NNTrainer] Current epoch: %d", epoch_idx);
```
Comment (jijoongmoon) [inline nntrainer/models/neuralnet.cpp:117]: I wonder, why do we need the log here? It means... everytime we call ```getCurrentEpoch()```, the log appear.
If application developer needs this, then he can generate the log in application. Not inside of NNTrainer.

File: nntrainer/models/neuralnet.cpp (hunk 1 above)
Comment (lhs8928) [reply nntrainer/models/neuralnet.cpp:792]: Changed to "stopped by stop callback function"

Comment (jijoongmoon) [reply nntrainer/models/neuralnet.cpp:792]: Great.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**